docs/aerodromes/classc/EastSale.md

```diff
@@ -4,7 +4,149 @@
 
 --8<-- "includes/abbreviations.md"
 
-Reserved.
+## Positions
 
-!!! note
-    Additional charts to the AIP may be found in the RAAF TERMA document, available towards the bottom of [RAAF AIP page](https://ais-af.airforce.gov.au/australian-aip){target=new}
\ No newline at end of file
+| Name               | Callsign           | Frequency        | Login Identifier             |
+| ------------------ | --------------     | ---------------- | -----------------------------|
+| East Sale ADC      | East Sale Tower    | 118.300          | ES_TWR                       |
+| East Sale SMC      | East Sale Ground   | 127.250          | ES_GND                       |
+| East Sale ACD      | East Sale Delivery | 134.100          | ES_DEL                       |
+| East Sale ATIS     |                    | 125.40           | YMES_ATIS                    |
+
+## Airspace
+ES ADC owns the Class C airspace from `SFC` to `A030`, within a 5nm radius of the ESL ARP. This airspace is designed to facilitate helicopter and circuit traffic.
+
+## Flight Category
+- Aircraft operating within ESL military airspace must be VFR unless otherwise indicated via flight plan or the pilot requests IFR or SVFR, due to weather.
+- Local area IFR operations should be minimised to those necessary, and aircraft must remain VFR until specifically cleared IFR by ATC. IFR clearance is a readback requirement.
+- **IFR Clearance Procedures**:
```

Comment:
Section can be removed. It's just describing how to give any normal IFR Clearance